packages/components/dl/dl-tokens.scss

```
@@ -0,0 +1,32 @@
+.kbq-description-list {
```

Должно соответствовать селектору в host.class

```suggestion
.kbq-dl {
```